civihr_default_theme/scss/pages/_my-leave.scss

```diff
@@ -9,4 +9,11 @@
   .button-container {
     margin-bottom: 40px;
   }
+
+  .btn-group {
+
+    > .btn {
+      border-color: transparent;
```

Comment:
This doesn't look right, why are you defining the border-color of a button inside a `btn-group`? This is obviously something that has to do with the style of the modal, right? So then why it is in the _my-leave page partial?